modular_skyrat/modules/bitrunning/code/virtual_domains/ancient_milsim/choice_beacon.dm

```diff
@@ -0,0 +1,83 @@
+/obj/item/choice_beacon/ancient_milsim
```

Comment:
Hey now this all seems like some **really** strong stuff to be dumping on **any** bitrunner domain they feel like.

Reply:
It's about on par with the Syndicate Assault items, -the only other PvP domain people'd really play-, so I don't think this'll be a problem; excluding the mech.

Reply:
No longer really strong but just strong.